modules/library/xml/src/test/java/org/geotools/xml/GMLParserTest.java

```diff
@@ -355,4 +365,47 @@ public void testProblemFeatures() {
             // fine, they were expected
         }
     }
+
+    @Test
+    public void testSecured() throws Exception {
```

Comment:
Would be nice to also have a test that passes because the parser only hits allowed resources (e..g, allow local file system, schemas.opengis.net, maybe xml.org). Exact same test code, just having a different mock checker setup.

Reply:
this is going to be troublesome on different OS

Reply:
Please clarify? You can get the absolute path of the file and replace it in the test XML for example.